itou/templates/nexus/dashboard.html

```diff
@@ -0,0 +1,742 @@
+                                                            aria-controls="pilotageInfoMore">
+                                                        <i class="ri-arrow-drop-right-line" aria-hidden="true"></i>
+                                                    </button>
+                                                </div>
+                                            </li>
+                                        {% endif %}
+                                        {% if Service.COMMUNAUTE not in activated_services %}
+                                            <li class="list-group-item list-group-item-action">
+                                                <div class="d-flex align-items-center gap-3">
+                                                    <div class="flex-grow-1 d-flex align-items-center gap-3">
+                                                        <img src="{% static_theme_images 'logo-communaute-inclusion-mono.svg' %}" height="35" alt="La Communauté de L'inclusion">
+                                                        <div>
+                                                            <strong class="d-block fs-sm">La Communauté de l’inclusion</strong>
+                                                            <p class="mb-0 fs-sm">
+                                                                Échangez avec d’autres structures et professionnels de l’insertion sur un espace d’entraide dédié.
+                                                            </p>
+                                                        </div>
+                                                    </div>
+                                                    <button class="btn btn-lg btn-ico-only btn-link stretched-link"
+                                                            type="button"
+                                                            aria-label="En savoir plus sur La Communauté de l’inclusion"
+                                                            data-bs-toggle="collapse"
+                                                            data-bs-target="#communauteInfoMore"
+                                                            aria-expanded="false"
+                                                            aria-controls="communauteInfoMore">
+                                                        <i class="ri-arrow-drop-right-line" aria-hidden="true"></i>
+                                                    </button>
+                                                </div>
+                                            </li>
+                                        {% endif %}
+                                    </ul>
+                                </div>
+                                <div class="col-12 col-xxl-7">
+                                    <hr class="mb-4 d-xxl-none">
+                                    <div id="nxAccordionInfoMore">
```

Comment:
```suggestion
                                    <div id="nxAccordionInfoMore" class="h-100">
```

Pour corriger [ce bug UI](https://www.notion.so/Page-d-accueil-2a15f321b604807aa0b3c5eda576aea3?d=2cb5f321b604802b9cc2001cb0815e2a&source=copy_link#2cb5f321b60480f49c9bf8961522ea70)